cli/mender-artifact/modify_existing_test.go

```diff
@@ -998,6 +998,7 @@ func TestModifyAllFlagsTested(t *testing.T) {
 	modifyWriteFlagsTested.addFlags([]string{
 		"ssh-args",
 		"version", // Could be supported, but we don't care about this.
+		"quiet", // Has no effect on the output
```

Comment:
This commit should be squashed together with the commit that adds the flag, so that we don't break git bisect.